hugr-py/src/hugr/std/int.py

```diff
@@ -45,9 +73,10 @@ class IntVal(val.ExtensionValue):
     """Custom value for an integer."""
 
     v: int
+    width: int = field(default=5)
 
     def to_value(self) -> val.Extension:
-        return val.Extension("int", INT_T, self.v)
+        return val.Extension("int", int_t(self.width), self.v)
```

Comment:
The parameter to this is meant to bre an index, not the width?

Reply:
That's `int_tv` (for integers with parametric size), this is calling `int_t` (a bit higher in the file).

Sorry for the messy diff, it's just a collection of changes I needed for guppy.
Though I didn't end up needing `int_tv`. I can drop it if it seems too confusing.

Reply:
Ah I see. Definitely drop `int_tv` if it's not used!